pkg/kvstore/client.go

```diff
@@ -43,7 +43,7 @@ func initClient(module backendModule, opts *ExtraOptions) error {
 
 	go func() {
 		err, isErr := <-errChan
-		if isErr {
+		if isErr && err != nil {
```

Comment:
Why is this change necessary?

Reply:
We should only fatal if `err != nil` there's no point in fatal if the channel is closed but there is no error.